src/providers/DevicesProvider/docs/AudioInputProvider.stories.mdx

```diff
@@ -17,7 +17,7 @@ The `AudioInputProvider` provides a list of the user's available `audio-input` d
   selectedDevice: string | null
 
   // The error when selecting device failed
```

Comment:
nit: 'The error if selecting a device fails'

Reply:
Oh, yes, I forget here, thanks Michael